src/Bridge/Symfony/Bundle/Action/SwaggerUiAction.php

```diff
@@ -95,6 +95,10 @@ public function __construct(ResourceNameCollectionFactoryInterface $resourceName
         $this->swaggerUiAction = $swaggerUiAction;
         $this->assetPackage = $assetPackage;
 
+        if (null === $this->twig) {
+            throw new \RuntimeException('You cannot use the "twig" service since the Twig bundle is not installed. Try running "composer require symfony/twig-bundle".');
```

Comment:
WDYT of adding a more user-related exception?
'The documentation cannot be displayed since the Twig bundle is not installed. Try running "composer require symfony/twig-bundle".'

Reply:
This is much better indeed! Thanks